cmd/kubefedctl/main.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-// kubefedctl is a tool for managing clusters in a federation.
+// kubefedctl is a tool for managing a KubeFed control plane.
```

Comment:
I'm not sure that kubefedctl manages the control plane? It is a tool for federation functions support.

Reply:
I think it's a reasonable generalization to say that kubefedctl manages a control plane given that it supports cluster registration, type enablement, and federation of resources.  In any case, what I have proposed is more accurate than the previous comment.